cleanup/bench_cleanup.py

```
@@ -0,0 +1,205 @@
+    return times, outs
+
+
+def bench_llm(samples: list[str], socket_path: str, *, max_tokens: int = 150,
+              timeout: float = 5.0) -> tuple[list[float], list[str]]:
+    times = []
+    outs = []
+    for s in samples:
+        t0 = time.perf_counter()
+        try:
+            out = send_request(socket_path, s, max_tokens=max_tokens, timeout=timeout)
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)